src/Microsoft.ML.Tokenizers.Data.Gpt2/Gpt2TokenizerData.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace Microsoft.ML.Tokenizers
+{
+    /// <summary>
+    /// Gpt2TokenizerData is internally used by Microsoft.ML.Tokenizers library to bind to the gpt2.tiktoken data file.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class Gpt2TokenizerData
```

Comment:
Does it need to be public?

Reply:
Good Question, let me try to make it internal and find out if the linker will complaint.

Reply:
Interesting, making the types internal worked fine. Even better!